vagrant-inventory

```diff
@@ -1,2 +1,3 @@
 [anxs]
-anxs.local ansible_ssh_host=192.168.88.22 ansible_ssh_port=22 ansible_ssh_user=vagrant ansible_ssh_private_key_file=~/.vagrant.d/insecure_private_key
+centos-vm ansible_ssh_host=192.168.88.22 ansible_ssh_port=22 ansible_ssh_user=vagrant ansible_ssh_private_key_file=~/.vagrant.d/insecure_private_key
```

Comment:
Is this related to the v10 update?  It looks interesting, but should be in a different Pull Request.

Reply:
I replaced this by vagrant-inventory/Vagrantfile of project's master branch.